public/js/profileView.js

```diff
@@ -54,6 +54,16 @@ window.addEventListener('load', async () => {
                             document.getElementById('phone_num').innerHTML = patient.phone;
                             document.getElementById('user_role').innerHTML = 'Patient';
                             document.getElementById('edit-profile').href = 'editPatientProfile.html';
+                            const resp = process(await fetch(`/patients/driver/${user.id.id}`));
+                            if(resp.ok) {
+                                const volDriver = await resp.json();
+                                volDriver.forEach((driver) => {
```

Comment:
Quick question, why do we need to iterate over each driver? Shouldn't there only be one?

Reply:
Yeah ideally, there is just one, but we made all endpoints deliver data in the same way so that it could simplify the process of reusing code and save time before milestone3 was due. This is a uniform way of doing it across all get requests so I just kept it the same

Reply:
Got it. I looked again at the route and it seems like the server indeed sends an array of drivers because the SQL statement is `SELECT * FROM driver`

Reply:
Yup! Sounds good